SphericalShell/README.md

```diff
@@ -0,0 +1,16 @@
+# Spherical Shell Problem ReadMe File
+
+Describes the basic parameters of the spherical shell/point source problem.
+
+## 1. Source Type
+* Point source of monoenergetic (14 MeV) neutrons at origin (0,0,0)
+* Thin spherical shell with r_in = 1000cm and r_out=1005cm, centered at origin (0,0,0)
```

Comment:
I think you misinterpreted one of my comments - this line still makes sense to leave in a section called geometry